scripts/qa_crowbarsetup.sh

```diff
@@ -4871,6 +4871,31 @@ zypper -non-interactive --gpg-auto-import-keys --no-gpg-checks install ses-upgra
     done
 }
 
+# Some resources are known to fail for a short time after Cloud6 deployment
+# because wicked ifreload call.
+# It's safe to clean existing errors now so we have a error-less crm status
+# output before the upgrade.
+function onadmin_cleanup_crm_resources
+{
+    local db_node=$(knife search node "roles:database-server AND pacemaker_founder:true" -a name | \
+        grep ^name: | cut -d : -f 2 | sed 's/\s//g')
+
+    if [ -n "$db_node" ] ; then
+        ssh $db_node "
+res=\$(crm resource list | grep vip-admin-database-default | cut -f 1 | sed 's/\s//g')
+crm resource cleanup \$res"
+    fi
+
+    local rabbit_node=$(knife search node "roles:rabbitmq-server AND pacemaker_founder:true" -a name | \
```

Comment:
This rabbit code is identical to the DB code so you should just put it inside `for svc in database rabbitmq`.